lib/worker/scrape.py

```diff
@@ -789,6 +926,82 @@ def _get_proxies(db):
         'https': piscina_url.value,
     }
 
+def _get_max_posts(db):
+    '''
+    Get a dictionary of max posts to scrape from the app configuration.
+    '''
+
+    max_posts_twitter = db.query(Configuration) \
```

Comment:
I added a function to Avatar's `model/configuration.py` module to simplify this because I noticed I was writing lots of boiler plate. See: https://github.com/TeamHG-Memex/avatar/commit/4acf56bb05714f3a193a9cc9c0e12a59ccea43cb#diff-2552471b8ff49cd85bb62b99eb431892